samples/bluetooth/channel_sounding_ras/reflector/README.rst

```diff
@@ -0,0 +1,79 @@
+.. _channel_sounding_ras_reflector:
+
+Bluetooth: Channel Sounding Reflector with Ranging Responder
+############################################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Channel Sounding Reflector with Ranging Responder sample demonstrates how to use the Ranging Service to provide ranging data to a client.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-sample-yaml::
+
+The sample also requires a device running a Channel Sounding Initiator with Ranging Requestor to connect to.
+
+Overview
+********
+
+The sample demonstrates a basic Bluetooth® Low Energy Peripheral role functionality that exposes the GATT Ranging Responder Service and configures the Channel Sounding reflector role.
+When Channel Sounding Ranging Data is generated by the controller, it will be forwarded to the Ranging Requestor Central via the Ranging Service.
+The Channel Sounding Ranging Data can then be used by the peer device to perform distance estimation.
+
+User interface
+**************
+
+The sample does not require user input and will advertise using the GATT Ranging Service UUID.
+The first LED on the development kit will be lit when a connection has been established.
+
+Building and running
+********************
+.. |sample path| replace:: :file:`samples/bluetooth/channel_sounding_ras/reflector`
+
+.. include:: /includes/build_and_run.txt
+
+Testing
+=======
+
+After programming the sample to your development kit, you can test it by connecting to another development kit that runs a Channel Sounding Initiator role with Ranging Requestor.
+
+1. |connect_terminal_specific|
+#. Reset the kit.
+#. Program the other kit with a Channel Sounding Initiator with Ranging Requestor sample.
```

Comment:
```suggestion
#. Program the other kit with the Channel Sounding Initiator with Ranging Requestor sample.
```
I guess we do not have such sample in NCS. I did not see one in Zephyr either. Link when there is such sample available.

Reply:
Will update when we have merged the initiator sample